docs/open-source-tizen/tools/mic/mic-reference.md

```diff
@@ -1,274 +1,249 @@
-
-- Livecd/liveusb
-
-  - Mainly used for an ia32 build, it can be burned to CD or usbstick, which can be booted into a live system or installation UI
-
+  - "Raw" format basically means hard disk dumping.
+  - It includes a partition table and all the partitions.
+  - The image is bootable directly.
+- Livecd and liveusb
+  - It is mainly used for an ia32 build.
+  - It can be burned to a CD or USB stick, which can be booted into a live system or installation UI.
 - fs
+  - "fs" means file-system.
+  - MIC can install all the Tizen files to a specified directory, which can be used directly as a chroot environment.
 
-  - “fs” means file-system
-  - mic can install all the Tizen files to the specified directory, which can be used directly as chroot env
+## Creating an Image<a name="create"></a>
 
-## Create image
-
-### Basic usage
-
-- Command line for image creation:
+To create a basic image, use the following command syntax:
 
 ```bash
-  $ mic create(cr) SUBCOMMAND <ksfile> [OPTION]
+$ mic create(cr) SUBCOMMAND <ksfile> [OPTION]
 ```
 
-- Sub-commands, to specify image format, include:
-
-```
-   help(?)            give detailed help on a specific sub-command
-   fs                 create fs image, which is also a chroot directory
-   livecd             create live CD image, used for CD booting
-   liveusb            create live USB image, used for USB booting
-   loop               create loop image, including multi-partitions
-   raw                create raw image, containing multi-partitions
-```
+In the command:
 
-- <ksfile>:
+- `SUBCOMMAND` specifies the image format:
 
-The kickstart file is a simple text file, containing a list of items about image partition, setup, Bootloader, packages to be installed, etc, each identified by a keyword.
+  ```bash
+  fs                 create fs image, which is also a chroot directory
+  livecd             create live CD image, used for CD booting
+  liveusb            create live USB image, used for USB booting
+  loop               create loop image, including multi-partitions
+  raw                create raw image, containing multi-partitions
+  ```
 
-In Tizen, the released image will have a ks file along with image. For example, you can download the ks file from: [http://download.tizen.org/releases/daily/trunk/ivi/latest/images/ivi-min...](http://download.tizen.org/releases/daily/trunk/ivi/latest/images/ivi-min-pc/ivi-min-pc-tizen_20120926.2.ks)
+- `<ksfile>` specifies the kickstart file.
+
+  The kickstart file is a simple text file, containing a list of items about, for example, image partition, setup, Bootloader, and packages to be installed. Each item is identified by a key word.
+
+  In Tizen, the released image has a `.ks` file along with the image. For example, you can download the `.ks` file from: [http://download.tizen.org/releases/daily/trunk/ivi/latest/images/ivi-min...](http://download.tizen.org/releases/daily/trunk/ivi/latest/images/ivi-min-pc/ivi-min-pc-tizen_20120926.2.ks)
```

Comment:
Link is broken

Reply:
It is legacy.  As I mentioned before, mic will be obsoleted.  Before then we just leave them.
Samsung doesn't take care of IVI profile.

Reply:
OK